tests/dhcp_relay/test_dhcp_relay_stress.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+import time
+
+from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory   # noqa F401
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_rand_selected_tor_m    # noqa F401
+from tests.dhcp_relay.dhcp_relay_utils import restart_dhcp_service
+from tests.common.helpers.assertions import pytest_assert, pytest_require
+from tests.common.utilities import wait_until
+from tests.ptf_runner import ptf_runner
+
+pytestmark = [
+    pytest.mark.topology('t0', 'm0'),
+    pytest.mark.device_type('vs')
```

Comment:
Please add this new script into PR test.

Reply:
You can add this script name into https://github.com/sonic-net/sonic-mgmt/blob/master/.azure-pipelines/pr_test_scripts.yaml, per to its support topology.

Reply:
Will add it later